.github/ISSUE_TEMPLATE/setuptools-warns-about-python-2-incompatibility.md

```diff
@@ -28,6 +28,7 @@ Your first course of action should be to reason about how you managed to get an
 <!-- These are the recommended workarounds for the issue. Please
 try them first. -->
 
+- [ ] Read [Python 2 Sunset docs](https://setuptools.readthedocs.io/en/latest/python%202%20sunset.html).
```

Comment:
I think the new doc filename should use hyphen instead of spaces to avoid URL escaping here.

Reply:
Acknowledged, but I prefer the spaces, especially as it appears nicely for users:

![image](https://user-images.githubusercontent.com/308610/82818959-8f3d3b00-9e6d-11ea-9c20-46c81e224d52.png)

Too many systems avoid spaces in filenames, even though they're well supported in almost all environments. I plan to continue to break the convention of replacing spaces with some other character, as spaces have their purpose.

Reply:
Please see PR https://github.com/pypa/setuptools/pull/2150.